src/Elastic.Markdown/Assets/main.ts

```diff
@@ -1,26 +1,12 @@
 // @ts-nocheck
-import "htmx.org"
+import htmx from "htmx.org"
 import "htmx-ext-preload"
 import {initTocNav} from "./toc-nav";
 import {initHighlight} from "./hljs";
 import {initTabs} from "./tabs";
 import {initCopyButton} from "./copybutton";
 import {initNav} from "./pages-nav";
 import {$, $$} from "select-dom"
-import { UAParser } from 'ua-parser-js';
-const { getOS } = new UAParser();
-
-document.addEventListener('htmx:beforeRequest', function(event) {
```

Comment:
This is no longer needed to support opening links in new tabs?

Reply:
https://github.com/elastic/docs-builder/pull/740